Platforms/QemuSbsaPkg/Docs/Features/feature_frontpage.md

```diff
@@ -0,0 +1,5 @@
+# Front Page
+
+Todo
```

Comment:
Todo

Reply:
Discussed offline, will resolve them when merging the documents into the common package.